libs/ui-libraries/src/lib/callback/callback.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { AuthService } from '@gc/core-data'
+
+@Component({
+  selector: 'vulnerability-checker-callback',
+  templateUrl: './callback.component.html',
```

Comment:
We don't need a template
```suggestion
  template: '',
```